tools/run_any.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2018 the Deno authors. All rights reserved. MIT license.
+"""
+gn can only run python scripts. This script launches any other executable.
+"""
+
+import util
+import sys
+
+util.run(sys.argv[1:], quiet=True, shell=False)
```

Comment:
Reuse `//third_party/v8/tools/run.py` ?

Reply:
I didn't know that that file existed. The files are eerily similar :)

It's a little weird to use some random v8 tool to build rust?
We could add a symlink from /tools/run.py to /third_party/v8/tools, which then requires some code in fix_symlinks.py.

I think this file is so simple that anything we do to reuse v8/tools/run.py is more complicated than just keeping two separate ones.

But I don't really care. Your call.

Reply:
I'd rather you just use `third_party/v8/tools/run.py` directly. It's been there forever, probably not going anywhere.

Reply:
Sure, done.